File: Data Structures & Algorithms/top-k-elements-in-list/submission-4.py
class Solution:
    def topKFrequent(self, nums: List[int], k: int) -> List[int]:
        dic={}
        for num in nums:
            dic.setdefault(num,0)
            dic[num]+=1
        eredmeny=[key for key,v in sorted(dic.items(),key=lambda x:x[1],reverse=True)[:k]]
        return eredmeny

            

File: Data Structures & Algorithms/duplicate-integer/submission-4.ts
class Solution {
    /**
     * @param {number[]} nums
     * @return {boolean}
     */
    hasDuplicate(nums: number[]): boolean {
        interface Dic{
            [key:number]:number
        }
        let dic:Dic={}
        for(let i=0;i<nums.length;i++){
            if (nums[i] in dic){
                return true
            }
            dic[nums[i]]=1
        
        }
        return false
    }
}
